lms/djangoapps/onboarding/admin.py

```diff
@@ -23,7 +23,10 @@
     OrganizationMetric,
     OrganizationMetricUpdatePrompt,
     MetricUpdatePromptRecord,
-    GranteeOptIn
+    GranteeOptIn,
```

Comment:
kindly order the imports alphabetically.

Reply:
done